AirCasting/CreateSessionViews/ConfirmCreatingSessionView.swift

```diff
@@ -93,7 +88,7 @@ struct ConfirmCreatingSessionView: View {
             ZStack {
                 if sessionContext.sessionType == .mobile {
                     GoogleMapView(pathPoints: [], isMyLocationEnabled: true)
-                } else {
+                } else if !sessionContext.isIndoor! {
```

Comment:
IDK about the force unwrap here. It will crash the value if not present... Can we add a fallback instead?
```Swift
} else if !(sessionContext.isIndoor ?? false) {
```